e2e-tests/functions

```diff
@@ -1691,6 +1727,92 @@ format_date() {
 	echo $(TZ=UTC $date -d@${timestamp} '+%Y-%m-%d %H:%M:%S')
 }
 
+get_bucket_name() {
+	local bucket_name=$1
```

Comment:
This variable should be `backup_name` given that following kubectl command, and how it is used in here `| $sed -e "s|BUCKET-NAME|$(get_bucket_name "$backup_name")|" \`

Reply:
fixed [6a9a86d](https://github.com/percona/percona-server-mongodb-operator/pull/1871/commits/6a9a86dc1765241623321dd84b3c3119083eb1f3)